src/main/flight/pid.c

```diff
@@ -281,32 +283,48 @@ void pidResetIterm(void)
     }
 }
 
+#ifdef USE_WING
+static float getWingTpaArgument(float throttle)
+{
+    const float sinPitchAngle = getSinPitchAngle();
+    const float pitchFactorAdjustment = scaleRangef(throttle, 0.0f, 1.0f, pidRuntime.tpaGravityThr0, pidRuntime.tpaGravityThr100);
+    const float pitchAngleFactor = sinPitchAngle * pitchFactorAdjustment;
```

Comment:
You can get rid of the `sinPitchAngle` variable and get the value directly
```suggestion
    const float pitchFactorAdjustment = scaleRangef(throttle, 0.0f, 1.0f, pidRuntime.tpaGravityThr0, pidRuntime.tpaGravityThr100);
    const float pitchAngleFactor = getSinPitchAngle() * pitchFactorAdjustment;
```

Reply:
i like one action per line, but ok :)